include/wx/textctrl.h

```diff
@@ -321,6 +329,13 @@ class WXDLLIMPEXP_CORE wxTextAttr
     void SetFontWeight(wxFontWeight fontWeight) { m_fontWeight = fontWeight; m_flags |= wxTEXT_ATTR_FONT_WEIGHT; }
     void SetFontFaceName(const wxString& faceName) { m_fontFaceName = faceName; m_flags |= wxTEXT_ATTR_FONT_FACE; }
     void SetFontUnderlined(bool underlined) { m_fontUnderlined = underlined; m_flags |= wxTEXT_ATTR_FONT_UNDERLINE; }
+    void SetFontUnderline(bool underlined, wxTextAttrUnderlineType type = wxTEXT_ATTR_UNDERLINE_SOLID, wxColour colour = *wxBLACK)
+    {
+        m_flags |= type;
```

Comment:
This cannot work. You are mixing `wxTextAttrFlags` and `wxTextAttrUnderlineType`. I suppose it has to be `wxTEXT_ATTR_FONT_UNDERLINE`?

Reply:
Still needs fixing.